### DISCUSSION TITLE
NLG TW Light Not working - Error: 'MAC no ack' (233)

### DISCUSSION THREAD

**Discussion by yonz2** [Koenkk/zigbee2mqtt#15769] [Devices]
Hi,
I've been battling with my new Paul Neuhaus (NLG) lights for some time now. They pair successfully with my zigbee2mqtt instance. After pairing, I am able to turn the light off and one for maybe 5 minutes, after that there is no connection to the device. I do, however see an error message on the zigee2mqtt logs:
```
Error 2022-12-27 12:58:10Publish 'set' 'state' to 'Esstisch' failed: 'Error: Command 0x90940300e00bfdb0/1 genOnOff.off({}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (Data request failed with error: 'MAC no ack' (233))'
Debug 2022-12-27 12:58:10Error: Command 0x90940300e00bfdb0/1 genOnOff.off({}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (Data request failed with error: 'MAC no ack' (233)) at ZStackAdapter.sendZclFrameToEndpointInternal (/app/node_modules/zigbee-herdsman/src/adapter/z-stack/adapter/zStackAdapter.ts:415:23) at Queue.executeNext (/app/node_modules/zigbee-herdsman/src/utils/queue.ts:32:32)
Error 2022-12-27 12:58:17Publish 'set' 'state' to 'Stehlampe-Wohnzimmer' failed: 'Error: Command 0xab910300e00bfdb0/1 genOnOff.off({}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (Timeout - 12009 - 1 - 64 - 6 - 11 after 10000ms)'
Debug 2022-12-27 12:58:17Error: Command 0xab910300e00bfdb0/1 genOnOff.off({}, {"sendWhen":"immediate","timeout":10000,"disableResponse":false,"disableRecovery":false,"disableDefaultResponse":false,"direction":0,"srcEndpoint":null,"reservedBits":0,"manufacturerCode":null,"transactionSequenceNumber":null,"writeUndiv":false}) failed (Timeout - 12009 - 1 - 64 - 6 - 11 after 10000ms) at Timeout._onTimeout (/app/node_modules/zigbee-herdsman/src/utils/waitress.ts:64:35) at listOnTimeout (node:internal/timers:559:17) at processTimers (node:internal/timers:502:7)
```
Here is a part of the [Zigbee Network Map](https://github.com/yonz2/aks44/blob/main/Zigbee-Network.png) showing the devices in question (as well as an IKEA Tradfri Lamp physically next to the "Esstisch" NLG Lamp")

Has anyone seen this before? Is there something I need to change inthe configuration (I assume not, as the "Support Status" attribute is set to "Supported")

BTW: I've tried this with a SONOFF ZBBridge (Flashed with Tasmota) as well, with exactly the same results.

(I just updated the firmware on the SONOFF Dongle to 20220219, still the same issue)

Setup:
```
Home Assistant 2022.12.8
Supervisor 2022.11.2
Operating System 9.4
Frontend 20221213.1 - latest

Zigbee2MQTT version [1.28.4]
Coordinator: Sonoff Zigbee 3.0 USB Dongle Plus
Coordinator type: zStack3x0
Coordinator revision: 20220219
Coordinator IEEE Address: 0x00124b002a2e87a3
```
Device Information (One of the two devices, the other one has exactly the same "symtpoms"
```
Friendly name: Esstisch
Description:
Last seen: N/A
Availability: [Disabled]
Device type: Router
Zigbee Model: NLG-TW light
Zigbee Manufacturer: Neuhaus Lighting Group
Description: Various tunable white lights - 2076-18
Support status: Supported

IEEE Address: 0x90940300e00bfdb0
Network address: 0x51BB
Firmware build date: 20210611
Firmware version: 1.0.6
Manufacturer: [Paul Neuhaus](https://www.zigbee2mqtt.io/supported-devices/#v=Paul%20Neuhaus)
Model: [NLG-TW light](https://www.zigbee2mqtt.io/devices/NLG-TW_light.html#paul%2520neuhaus-nlg-tw%2520light)
Power: Mains
Interview completed: True
```

**Comment by haraldknoll** [comment]
Hi,

same here. I run it with deCONZ. After power on, all light stay active for minutes, 1 hour or a couple of days. It follows no pattern, but its allways a Paul Neuhaus light. Paul Neuhaus does not provide any firmware update. BTW it happens with Hue bridges as well. I play now arround with Paul Neuhaus lights for years and the have radom "turn on´s" and loss of communication. I know the PM, but it is impossible to get ever a FW update. Old Paul Neuhaus light can not be updated, the new ones can and regsiter at the OTA server. I cant say what i is, but it seems routing of this types of lamps is faulty. No issues with other verdors - lamps stay active for years and route correctly.

![Screenshot 2023-03-08 212700](https://user-images.githubusercontent.com/68897141/223842157-f83131c7-d16c-42b8-a5a5-0cb92897b6f0.png)

**Reply by yonz2** [reply]
Did you see my comment on another Topic:

> Update:
> Turned out that this is a coexistence problem btw. NLG Lamps and Apple Homepod Mini. The Homepod Mini broadcasts a message to all PANs (Destination PAN: 0xFFFF) ca. every 5 minutes. All other ZIGBEE devices ignore this, except the NLG Lamps, they simply freeze. The only way around this is then to unplug the Apple Homepod Mini, and re-join the NLG lamps to the Zigbee Network. (It is then a question of Apple Homepod Mini or NLG Lamps, only one of them works at the same time)
> 

I did a trace (Wireshark with a China Dongle) to capture everything, and it was so obvious that as soon as the HomePod Mini came online, the NLG Lamps stopped working. This is, without a doubt, a problem with the NLG Firmware. (All other devices, incl. Cheap China-Stuff works 100%, only the "expensive" NLG Lamps don't

(See Issue: https://github.com/Koenkk/zigbee2mqtt/issues/15865 )
(And Discussion Topic: https://github.com/Koenkk/zigbee2mqtt/discussions/16257 )

BR
Yonz

**Reply by haraldknoll** [reply]
Thanks for the update! I dont have a Apple device in the network, but maybe a other device which sends Destination PAN: 0xFFFF. Have to check. However, the  NLG lamp dont freeze fully (yes they do in terms of zigbee routing), but they stop routing and dont respont to unicast packets. However they respond to broadcasts even if not reachable.

**Reply by yonz2** [reply]
Exactly, that's the same behaviour I observed. I was able to control the lights with the Standard Remote Control; but not via Zigbee. After I disconnected the Apple Homepod everything is fine.
I have open three tickets via the NLG Support (on their Website) the first response was "Read the f..ing manual" (or something along those lines) For the 2nd and 3rd Ticket I haven't heard anything from them. I tried E-Mail but no response. 

mfg
Yonz

**Comment by budachst** [comment]
Hi,

I do have two different NLGs, one of those double ring LEDs and a ceiling lamp using a slightly different version of the receiver (without IR), but with a newer FW. The receiver using the 1.0.6 FW behaves just the way described here. I don't have an Apple iPod, but an Apple TV.

However, the ceiling lamp's receiver does has a newer FW 1.3 and that one does work flawlessly. Since the rest of the electrical setup is the same - looks looks like the same power supply, you can also interchange these receivers. Does anyone know of an option to maybe flash the new FW to the receiver using the 1.0.6 one?

And yes, the guys at Neuhaus, simply tend to ignore anything you throw at them. My support case has now been closed twice as resolved, even though they didn't even get back to me.

**Comment by budachst** [comment]
Jfyi - I actually called NLG on Monday and discussed this issue with the support rep. He agreed to swap the old receiver with a new one. That was actually easier than I expected this time, as there seems to have been a change in personnel at the support hotline and the rep was very receptive. As soon, as I got my hands onto the new receiver, I will swap it and see, if this issue is gone. If that's the case, maybe you guys also want to take a shot at getting your receivers swapped.

**Reply by haraldknoll** [reply]
I actually had a similar discussion with a member of the board. He agreed to swap the Zigbee module. I have more than 60 lamps from NLG and 30 alone in my office. Some are 6 meters on the ceiling. Not fun to replace. A firmware upgrade would be easier. However, I am glad to hear a response from Paul Neuhaus, because non responding routers cause random light “on” and other troubles in the Zigbee network. I struggle with this for years now.

**Reply by yonz2** [reply]
Thanks for the update. It was on my to-do list to try to call them. If you do get the new receiver, please post the results here as well.

**Reply by budachst** [reply]
Got the new receiver today and just only switched the actual receiver - reset the controller using the power on/off sequence and had it discovered by my HUE Bridge.  The receiver has the same FW 1.3 as the other one, which is working fine and so is the replacement one.
